MapleServer2/PacketHandlers/Game/StatPointHandler.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Extensions.Logging;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Servers.Game;
+using MapleServer2.Packets;
+using System;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class StatPointHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.STAT_POINT;
+
+        public StatPointHandler(ILogger<StatPointHandler> logger) : base(logger) { }
+
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            byte mode = packet.ReadByte();
+
+            switch (mode)
+            {
+                case 2: 
```

Comment:
Can we use enums for these modes?

Reply:
I think it does add a layer of readability but I'm not sure it's needed here, especially since the RECV/SEND modes do not match (e.g. mode 3 in RECV corresponds to sub-op 1 in SEND).

Reply:
I think in almost all cases the RECV and SEND will not match, and on top of that, sometimes you have to send multiple SEND packets for a single RECV packet. It's pretty much just for readability and maintainability

Reply:
Enums added in branch stat-points-3